app/routers/users.py

```
@@ -0,0 +1,65 @@
+    return
+
+@router.patch("/me", summary="프로필 수정", description="닉네임, 프로필 이미지, 비밀번호 변경", tags=["users"])
+async def patch_user(request: UserLoginRequest):
+    return
+
+@router.delete("/me", summary="회원 삭제", description="계정을 삭제합니다.", tags=["users"])
+async def delete_user(request: UserLoginRequest):
+    return
+
+@router.get("/me", summary="내 프로필 조회", description="로그인한 사용자 본인 정보를 조회합니다", tags=["users"])
+async def get_me(request: UserLoginRequest):
```

Ruff (0.14.11)

Unused function argument: `request`

(ARG001)